### ISSUE TITLE
Use query, 

### ISSUE THREAD

**Issue by hubgit** [wesleytodd/express-openapi#22]
The generic TypeScript description for `express.RequestHandler` allows type definitions for the request path params, response body, request body and request query params:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/03fa4ecc7ecbe94e50b5e5fe03d7add64a198198/types/express-serve-static-core/index.d.ts#L53-L66

Would it be possible to make use of these when generating the OpenAPI description for a route?

**Comment by wesleytodd** [comment]
Yes, I do think that is the right way to do this.  Happy to accept a PR for this.
